tests/test_api/api_test_utils.py

```diff
@@ -62,20 +62,7 @@ def api_get_course(client, term_id, section_id, expected_status_code=200):
     return response.json
 
 
+@cachify('instructor_uids_{section_id}_{term_id}')
 def get_instructor_uids(section_id, term_id):
-    sql = """
-        SELECT DISTINCT instructor_uid
-        FROM sis_sections
-        WHERE
-            term_id = :term_id
-            AND section_id = :section_id
-            AND instructor_uid IS NOT NULL
-    """
-    rows = db.session.execute(
-        text(sql),
-        {
-            'section_id': section_id,
-            'term_id': term_id,
-        },
-    )
-    return [row['instructor_uid'] for row in rows]
+    course = SisSection.get_course(section_id=section_id, term_id=term_id)
+    return [instructor['uid'] for instructor in course['instructors']]
```

Comment:
This is an unrelated fix. If the course is cross-listed then the SQL query will not pick up instructors associated with _other_ section_ids. Thus, I swapped in the proper `SisSection.get_course(...)`.